frame/ibc/src/weights.rs

```diff
@@ -55,7 +55,7 @@ impl<T: frame_system::Config> crate::weight::WeightInfo for WeightInfo<T> {
 	// Storage: Ibc ClientUpdateTime (r:0 w:1)
 	// Storage: Ibc ClientUpdateHeight (r:0 w:1)
 	fn update_client() -> Weight {
-		(473_000_000 as Weight)
+		(498_000_000 as Weight)
```

Comment:
jesus

Reply:
Must be the signature verification in tendermint-rs, and it’s still just two signatures that were used.